### PR TITLE
dot1x tests with chewie v0.0.5 (radius & state handlers).

### PR DESCRIPTION
**Pull request by Bairdo**
Tests check all dot1x prometheus variables.

Tests must run serial as cannot configure chewie's radius port

### CODE REVIEW

**Comment by Bairdo** [comment]
Dont merge yet, let me see what was wrong with that chewie revert https://github.com/faucetsdn/chewie/pull/19

File: tests/integration/mininet_tests.py [hunk 1]
```diff
@@ -155,7 +156,7 @@ def test_untagged(self):
         self.verify_events_log(event_log)
 
 
-class FaucetUntagged8021XSuccessTest(FaucetUntaggedTest):
+class FaucetSingle8021XSuccessTest(FaucetUntaggedTest):
```
Comment (Bairdo) [inline tests/integration/mininet_tests.py]: Doesn't follow the usual naming Format, but for some reason if the name was FaucetSingleUntagged8021XSuccessTest the test fails. Also tried a few others and seems to be based on length. Haven't looked into why yet

Comment (Bairdo) [reply tests/integration/mininet_tests.py]: chewie never seems to receive any EAP packets.

```
test_untagged (mininet_tests.FaucetSingleUntagged8021XFailureTest) ... Aug 29 01:40:19 faucet INFO     Reloading configuration
Aug 29 01:40:19 faucet INFO     configuration /var/tmp/faucet-tests-l4n7a0z3/FaucetSingleUntagged8021XFailureTest-test_untagged/faucet.yaml changed, analyzing differences
Aug 29 01:40:19 faucet INFO     Add new datapath DPID 4256019433 (0xfdadb3e9)
Aug 29 01:40:19 faucet.valve INFO     DPID 4256019433 (0xfdadb3e9) faucet-1 table ID 0 table config dec_ttl: None exact_match: False match_types: {('eth_type', False), ('eth_src', False), ('in_port', False)} meter: False miss_goto: None name: port_acl output: True set_fields: () size: 32 vlan_port_scale: None
Aug 29 01:40:19 faucet.valve INFO     DPID 4256019433 (0xfdadb3e9) faucet-1 table ID 1 table config dec_ttl: None exact_match: None match_types: (('eth_dst', True), ('eth_type', False), ('in_port', False), ('vlan_vid', False)) meter: None miss_goto: None name: vlan output: True set_fields: ('vlan_vid',) size: 32 vlan_port_scale: 1.1
Aug 29 01:40:19 faucet.valve INFO     DPID 4256019433 (0xfdadb3e9) faucet-1 table ID 2 table config dec_ttl: None exact_match: None match_types: (('eth_dst', True), ('eth_src', False), ('eth_type', False), ('in_port', False), ('vlan_vid', False)) meter: None miss_goto: eth_dst name: eth_src output: True set_fields: ('vlan_vid', 'eth_dst') size: 32 vlan_port_scale: 4.1
Aug 29 01:40:19 faucet.valve INFO     DPID 4256019433 (0xfdadb3e9) faucet-1 table ID 3 table config dec_ttl: None exact_match: True match_types: (('eth_dst', False), ('vlan_vid', False)) meter: None miss_goto: flood name: eth_dst output: True set_fields: None size: 32 vlan_port_scale: 4.1
Aug 29 01:40:19 faucet.valve INFO     DPID 4256019433 (0xfdadb3e9) faucet-1 table ID 4 table config dec_ttl: None exact_match: None match_types: (('eth_dst', True), ('in_port', False), ('vlan_vid', False)) meter: None miss_goto: None name: flood output: True set_fields: None size: 32 vlan_port_scale: 2.1
Aug 29 01:40:19 faucet INFO     dot1x enabled on faucet-1 Port 1, NFV interface uac4-eth0
Aug 29 01:40:19 faucet INFO     Starting
Aug 29 01:40:19 faucet INFO     Radius Listening on 0.0.0.0:0
Aug 29 01:40:19 faucet INFO     waiting for eap.
Aug 29 01:40:19 faucet INFO     waiting for radius.
Aug 29 01:40:19 faucet.valve INFO     DPID 4256019433 (0xfdadb3e9) faucet-1 Cold start configuring DP
Aug 29 01:40:19 faucet.valve INFO     DPID 4256019433 (0xfdadb3e9) faucet-1 Configuring VLAN 100 vid:100 ports:Port 1,Port 2,Port 3,Port 4
Aug 29 01:40:19 faucet.valve INFO     DPID 4256019433 (0xfdadb3e9) faucet-1 Port 1 (b1 - 802.1x client.) configured
Aug 29 01:40:19 faucet.valve INFO     DPID 4256019433 (0xfdadb3e9) faucet-1 Port 2 (b2) configured
Aug 29 01:40:19 faucet.valve INFO     DPID 4256019433 (0xfdadb3e9) faucet-1 Port 3 (b3) configured
Aug 29 01:40:19 faucet.valve INFO     DPID 4256019433 (0xfdadb3e9) faucet-1 Port 4 (NFV host - interface used by controller.) configured
Aug 29 01:40:22 faucet.valve INFO     DPID 4256019433 (0xfdadb3e9) faucet-1 L2 learned 62:a6:95:75:a3:da (L2 type 0x86dd, L3 src fe80::60a6:95ff:fe75:a3da, L3 dst ff02::2) Port 2 VLAN 100  (1 hosts total)
Aug 29 01:40:22 faucet.valve INFO     DPID 4256019433 (0xfdadb3e9) faucet-1 L2 learned de:da:b8:c8:68:47 (L2 type 0x86dd, L3 src fe80::dcda:b8ff:fec8:6847, L3 dst ff02::2) Port 3 VLAN 100  (2 hosts total)

```

**Comment by anarkiwi** [comment]
Would you update to use Chewie 0.0.5 please? I messed up the release of .3/4.

**Pull request merged**